jetty-core/jetty-http3/jetty-http3-common/src/main/java/org/eclipse/jetty/http3/HTTP3Session.java

```diff
@@ -666,7 +667,7 @@ public void inwardClose(long error, String reason)
             }
         }
 
-        failStreams(stream -> true, error, reason, true);
+        failStreams(stream -> true, error, reason, true, new IOException(reason));
```

Comment:
Shouldn't this be a `QuietException` too?

Reply:
No because the quiet exceptions are used when receiving an event, while here it's generating the event.